generics/services/sunbird.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * name : sunbird.js
+ * author : Vishnudas
+ * Date : 07-Mar-2022
+ * Description : All Sunbird learner related api call.
+ */
+
+//dependencies
+
+
+const request = require('request');
+const sunbirdBaseUrl = process.env.SUNBIRD_SERVICE_URL;
```

Comment:
@VISHNUDAS-tunerlabs add this env value in envVariables.js file in the root directory. and make it required true